Tools/autotest/arducopter.py

```diff
@@ -7449,14 +7449,29 @@ def ParameterValidation(self):
             "MOT_PWM_MAX": 50,
         })
         self.drain_mav()
-        self.assert_prearm_failure("Check MOT_PWM_MIN/MAX")
+        self.assert_prearm_failure("Motors: Check MOT_PWM_MIN and MOT_PWM_MAX")
         self.progress("invalid; min must be less than max (equal case):")
         self.set_parameters({
             "MOT_PWM_MIN": 100,
             "MOT_PWM_MAX": 100,
         })
         self.drain_mav()
-        self.assert_prearm_failure("Check MOT_PWM_MIN/MAX")
+        self.assert_prearm_failure("Motors: Check MOT_PWM_MIN and MOT_PWM_MAX")
+        self.progress("Spin min more than 0.3")
```

Comment:
This can be
```suggestion
        self.start_subtest("Spin min more than 0.3")
```

but it's no big thing.